style.css

```diff
@@ -104,6 +104,222 @@
   list-style: none;
 }
 
+/* work-section */
+
+.work-section {
+  width: 100%;
+  height: 2788px;
+  top: 718px;
+}
+
+.work-container {
+  display: grid;
+  grid-template-columns: auto;
+  padding: 16px, 6px;
+  border-radius: 16px;
+  margin: 36px;
+  border: 1px solid #dfe1e6;
+}
+
+.cards {
+  display: flex;
+  height: 100%;
+  width: 100%;
+  flex-direction: column;
+  padding-left: 24px;
+}
+
```

Comment:
- [ ] I noticed that you are using both, `grid` and `flex` for styling individual project cards. However, as given in the project requirements, please consider using `display: flex` for styling individual cards and `display: gird` for the section that contains all the project cards 🛠🤓 Following is a quote for a related requirement 👇

  > _In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). **Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.**_


- [ ] I noticed that the image is moving out 😓 I think this is because of the `padding-left` you are applying for `.cards` class. Please fix this as mentioned in the Figma design. This can be done by removing `padding-left` from `.cards`  and implementing following 👇 _However, please note that this is just an example, you can use your own styles_

```
.cards {
  padding: 1rem;
  align-items: center;
  // other properties
}

.works-container {
  display: flex;
  width: 90vw;
  margin: 0 auto;
  // other properties
}
```
- [ ] Similarly, please make sure to add some padding to the top and bottom of the project card, and align the project technologies (html, css, javascript) and text as mentioned in the image below 🤓⛏

Your design | Figma design
---| ---|
![image](https://user-images.githubusercontent.com/106119105/235188721-f82f43cf-41eb-4246-add0-39c8e6a85500.png) | ![image](https://user-images.githubusercontent.com/106119105/235187686-2ca7211e-77b2-41f1-b8ac-35b2f47a6e25.png)